docs/src/prototypes/meetingOptions/components/MSTeamsFooter.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Divider, Layout } from '@stardust-ui/react'
+
+export default props => {
+  return (
+    <div style={{ position: 'fixed', width: 'inherit', bottom: '0px' }}>
```

Comment:
raises couple of concern points (should be mentioned in the PR description maybe):
- how we could introduce these styles as part of theme?
- `Layout` alignment options are limited

Reply:
Will mention them.